requirements-dev.txt

```diff
@@ -6,7 +10,8 @@ Sphinx==1.8.0
 sphinx-autobuild==0.7.1
 sphinxcontrib-fulltoc==1.2.0
 sphinxcontrib-napoleon==0.7
-flake8==3.8.*
-yapf==0.30.*
 nbsphinx==0.7.1
 coverage==5.2.*
+
+# optional dependencies
+orjson==3.5.2
```

Comment:
Do we need this in the dev dependencies, since it's included in one of the extras? 

If it's included in the dev dependencies it will mean that the `if orjson is not None` checks on `stac_io` are always going to come back `True` for developers who set up their dev environment using this file, regardless of whether they install the package using `[orjson]` or not, which could cause some confusion.

Reply:
If it's not in the dev environment, Pylance complains about the package not existing in the virtual env. I think perhaps the approach here would be to have the dev environment have all optional dependencies, and then write CI tests that explicitly check against the case where the dependencies aren't installed. What do you think?

Reply:
That seems like a good approach to me.